src/views/htmlcontent/src/js/constants.ts

```diff
@@ -1,19 +1,24 @@
 /** Results Pane Labels */
-export const maximizeLabel = 'Maximize';
-export const restoreLabel = 'Restore';
-export const saveCSVLabel = 'Save as CSV';
-export const saveJSONLabel = 'Save as JSON';
-export const saveExcelLabel = 'Save as Excel';
-export const resultPaneLabel = 'Results';
-export const selectAll = 'Select all';
-export const copyLabel = 'Copy';
-export const copyWithHeadersLabel = 'Copy with Headers';
+export let maximizeLabel = 'Maximize';
+export let restoreLabel = 'Restore';
+export let saveCSVLabel = 'Save as CSV';
+export let saveJSONLabel = 'Save as JSON';
+export let saveExcelLabel = 'Save as Excel';
+export let resultPaneLabel = 'Results';
+export let selectAll = 'Select all';
+export let copyLabel = 'Copy';
+export let copyWithHeadersLabel = 'Copy with Headers';
 
 /** Messages Pane Labels */
-export const executeQueryLabel = 'Executing query...';
-export const messagePaneLabel = 'Messages';
-export const lineSelectorFormatted = 'Line {0}';
-export const elapsedTimeLabel = 'Total execution time: {0}';
+export let executeQueryLabel = 'Executing query...';
+export let messagePaneLabel = 'Messages';
+export let lineSelectorFormatted = 'Line {0}';
+export let elapsedTimeLabel = 'Total execution time: {0}';
 
 /** Warning message for save icons */
-export const msgCannotSaveMultipleSelections = 'Save results command cannot be used with multiple selections.';
+export let msgCannotSaveMultipleSelections = 'Save results command cannot be used with multiple selections.';
+
+export let loadLocalizedConstant = (key: string, value: string) => {
+    this[key] = value;
```

Comment:
Minor: maybe document what this does / how it works. It's kinda magic (but cool)